index.html

```diff
@@ -4,6 +4,15 @@
 permalink: /
 ---
 
+<head>
```

Comment:
This isn't supposed to be added here. The head tag is already defined in [head.html](/COPS-IITBHU/cops-website/blob/master/_includes/head.html) and [index_head.html](/COPS-IITBHU/cops-website/blob/master/_includes/index_head.html) files. So CSS will be included there.